MovieAppPetProject/MovieAppPetProject/SearchMovies/SearchMovies-ViewController.swift

```diff
@@ -6,56 +6,62 @@
 //
 
 import UIKit
-import SDWebImage
 
 class SearchMoviesViewController: UIViewController, UITableViewDelegate, UITableViewDataSource {
-    
-    
+
     private let searchMoviesViewModel = SearchMoviesViewModel()
```

Comment:
no repo being injected here or tablw view?😄

Reply:
injecting repo into view controller? 
would you mind to please explain why that is needed?

Reply:
I think I'm not understanding is why I'm asking 🤝

Reply:
private let searchMoviesViewModel = SearchMoviesViewModel(delegate: self)
When you do this, you can get rid of the dispatchQueue fro,m the other pr